app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,38 @@
+@import "materialize/components/color";
+$primary-color: color("blue", "lighten-2") !default;
+@import "materialize";
+
+$mainText: #e8eaf6; 
```

Comment:
Name of variable `mainText` should be written in all lowercase letters with hyphens instead of underscores